Add work portfolios with basic styling

BenMKT opened the pull request:
Added several work projects to my portfolio with basic styling by building the middle section of the mobile website.

Trast00 commented on styles.css:
- [ ] Kindly consider adding button interaction like active and pressed. You can do that by using pseudo-class like `: active` `: enabled`. This will make your website way more user friendly 😄 
**Make sure to copy the figma template**
![image](https://user-images.githubusercontent.com/74411135/227716107-209309f5-4173-4075-8e2a-3fbf5982cecd.png)

Trast00 commented on index.html:
- [ ] Please make sure the list "JavasScript", "HTML" and "CSS" are a list `ul` and each of them a `li`. This will make you website way more accessible (ex: for screen readers) and user friendly 😃

Trast00 commented on index.html:
- [ ] Please, make sure the about section has the same background as in the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=mnapuNy9Wgx1RKyE-0). This will make your project way more good looking (as the design are done by professional) which will improve the user experience.

Your about section has currently no styling, but it should be styled. You can do that by changing the background color and the `border-raduis-top-left`
**you're about section**
![image](https://user-images.githubusercontent.com/74411135/227716339-46bdb788-3179-45b2-85aa-af8a9e84ace3.png)

**The about section in the figma template**
![image](https://user-images.githubusercontent.com/74411135/227716405-31b74a2b-fd07-43cb-8f2e-3a0e3278a485.png)

Trast00 commented on index.html:
- [ ] Please, make sure the about section has the same background as in the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=mnapuNy9Wgx1RKyE-0). This will make your project way more good looking (as the design are done by professional) which will improve the user experience

The work sections seem not have any background, it should have a background color of `#E5E5E5;` which is a bit darker.
**You work section**
![image](https://user-images.githubusercontent.com/74411135/227716565-29b01841-b19f-4c8f-b134-5f2eb2c311ff.png)


**Figma template work section**
![image](https://user-images.githubusercontent.com/74411135/227716547-cb7dd848-5699-432b-b93c-7bd07d8a6e2c.png)

Trast00 reviewed (changes requested):
# Require Change ♻️ 
Hi @BenMKT , 

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![almost-there-reach](https://user-images.githubusercontent.com/74411135/225062441-793da0f4-3ef1-4d98-bee9-9c200c7121a7.gif)

## Highlights
- Impressive clean code  🧼 😃 
- The project look almost same as the figma template 😮 🎖️ 
- Professional documented project 📖 

## Required Changes ♻️
_Check the comments under the review._

- Please consider implementing all the change I left below (inline comment) 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@Trast00) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BenMKT commented on index.html:
Added the background color  #E5E5E5 as in Figma to my 'about' class in the CSS doc to resolve issue .

BenMKT commented on index.html:
- Inserted the background color to the about class in CSS.
- Gave it a border-radius-top-right in the container class in the CSS file to resolve this.

BenMKT commented on index.html:
- inserted the <ul> and <li> tags to my HTML file to resolve this.

BenMKT commented on styles.css:
Have utilized ":active" and ":hover" pseudo-classes in my CSS file to add friendlier button UI/UX to resolve this.

BenMKT commented on styles.css:
- Added image margin property to the CSS  ".image1" class selector to resolve this bug.
- Centered all project cards using a CSS flex-item "align-self" property in my CSS file to resolve this bug.

BenMKT commented on styles.css:
- Fixed bug by modifying the background-size property from "auto" to "contain" in my CSS body class selector.

Strangeal reviewed (changes requested):
Hi @BenMKT, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![Alt Text](https://media.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif)

 ## Required Changes ♻️
- [ ] Please make sure the background color you added to the work section container spans across the full width of the week section.
<img width="594" alt="Screenshot 2023-03-25 at 05 13 51" src="https://res.cloudinary.com/dxsom7jmx/image/upload/v1679759634/Screenshot_2023-03-25_at_08.42.50_y1ojzl.png">

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BenMKT commented:
- Added a relative full-width length value of  100% to the "works" main section to make sure the section and all its associated styles span the full width of its parent container regardless of viewport size to resolve this bug.

amon-cofie reviewed (changes requested):
Hi @BenMKT , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

🚀 No linter errors
🚀 Descriptive pull request
🚀 Good commit messages

 ## Required Changes ♻️

I can see that you made a huge effort in trying to resolve the issue that the previous reviewer pointed out. Unfortunately, however, the problem still persists: 

![image](https://user-images.githubusercontent.com/107795045/227732739-05d17fac-a5c9-4382-b7af-50e639ca1674.png)

I'm happy to help you figure out how to resolve this problem. You can contact me on slack so we set up a meeting. 👍 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BenMKT commented:
- Removed padding from the body selector which solved the background-color bug.
- Removed margin from the cards solving the screen fit bug.
- These are all changes suggested by code reviewers.

joskalenda reviewed (approved):
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @BenMKT , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
| <img src="https://media2.giphy.com/media/2NpKgBuUZtXpu/giphy.gif"/> |

### Highlights
✅ ... Good design
---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

usorfaitheloho commented on index.html:
You did well with this project and kudos to you 😄 . Just a reminder, please you're to have only one `h1` tag in your project to enhance search engine optimization and accessibility, so kindly change it to a lower `hx` tag 👍

usorfaitheloho commented on index.html:
The list of technologies should be listed in a `<ul> <li> </li></ul> ` tag and not you having different ul's and li's yag in between .Kindly implement this change so you have a clean code that also follows semantics purposes 😄

usorfaitheloho commented on styles.css:
Kindly implement the following design changes for your `works` section, so it shows that you have a keen attention to detail and it makes your project look better 👍 

1. please use the right font color for your card header eg Tonic, which is font color `#172B4D`
2. Kindly give a `font-weight` property of value `600` for the subtitles of your card so it looks better and close to the design

figma design :
![image](https://github.com/BenMKT/myPortfolio/assets/20537712/08fd8f89-f1e5-4af1-b905-a92fd4b7f992)

Your design :
![image](https://github.com/BenMKT/myPortfolio/assets/20537712/2648f1d8-a7d7-4236-b6da-ba3e837b08b0)

usorfaitheloho commented on styles.css:
Kindly implement these changes for the works section :

1. The `Languages`, `Framework` and `skills` should be in  a bolder font style as seen in the design given
2. the `lanhguages` are lsited ie we jhave their names and the icons as well but in your design we have only the icons, please fix this and resubmit 😄 

Figma design :
![image](https://github.com/BenMKT/myPortfolio/assets/20537712/dcfeede3-5b21-4f25-8dae-42ad184889db)

Your design :

![image](https://github.com/BenMKT/myPortfolio/assets/20537712/a2aff4a0-bd43-4fcb-bf2e-1b932238c47c)

usorfaitheloho commented on index.html:
[Optional] The headline section of your project lacks some background images which made it look a bit scanty, please you could try to see if you can add them as well, although its not part of this milestone but it would be nice if you add them 😄

usorfaitheloho reviewed (changes requested):
> Hi @BenMKT ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

 ![](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966) [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Design followed CSS requirement of Grid and Flexbox ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BenMKT commented on index.html:
- Removed nested `ul` tags with their accompanying classes and styling for cleaner code in About section.

BenMKT commented on index.html:
- Changed `About Myself` heading to lower `h2` for cleaner code standards.

BenMKT commented on styles.css:
- Added color `#172b4d` to all my cards' `h2` headers to conform to Figma design.
- Added a `font-weight` property with a value of 600 to all my cards' subtitles on the `flexy` class. Also added the `font-size` property.

BenMKT commented on styles.css:
- Added `font-weight` property to the `list` class value to conform to Figma design.
- Added names to the language icons with their relevant styles to conform to the Figma design.

BenMKT commented on index.html:
- Being optional will enforce later on after consultations as trying to add but distorting whole design

Efiamotu-1 reviewed (approved):
Hi @BenMKT , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

![giphy](https://user-images.githubusercontent.com/89055502/234114221-4ca295de-fd8f-4af6-9c49-a6d4c6ea67f7.gif)


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit history :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [![@Efiamotu-1](https://img.shields.io/github/followers/Efiamotu-1?label=Habeeb-Efiamotu-Musa&style=social)](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[![Habeeb Efiamotu Musa](https://img.shields.io/badge/linkedin-@Habeeb-crimson)](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [![@EFYAMOTU](https://img.shields.io/twitter/follow/omarramoun?label=Habeeb-Efiamotu-Musa&style=social)](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [![Habeeb Efiamotu Musa ](https://img.shields.io/badge/slack-@Habeeb-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
